github_api_helper.js

```diff
@@ -98,22 +98,48 @@ function getConclusion (annotations) {
  * @param {import('probot').Context} context Probot context
  * @param {Number} runID check run identifier
  * @param {Object} output merged scan output
+ * @param {Number} annotationsPerPage Optional: number of annotations to send with one API call.
  * @returns {Promise<any>} GitHub response
  * See: https://developer.github.com/v3/checks/runs/#update-a-check-run
  */
-function sendResults (context, runID, output) {
+function sendResults (context, runID, output, annotationsPerPage) {
   const { owner, repo } = context.repo()
-  const completedAt = new Date().toISOString()
+  let numAnnotationsLeftToSend = annotationsPerPage || output.annotations.length
+  if (numAnnotationsLeftToSend <= 0 || numAnnotationsLeftToSend > 50) {
+    numAnnotationsLeftToSend = 50
```

Comment:
This arbitrary number of `50` here could use a constant instead.  You use it multiple times too.

Reply:
Yep, I will change that.

Reply:
Yes, I just changed that.